src/file/file.model.ts

```diff
@@ -3,6 +3,7 @@ import { ServerError } from '../utils/errors/application.error';
 import { IFile } from './file.interface';
 import { KeyAlreadyExistsError } from '../utils/errors/client.error';
 import { MongoError } from 'mongodb';
+import { NextFunction } from 'connect';
 
 const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
Don't import the whole mongoose package just for ObjectId and other parts of the package.